start-demo.sh

```diff
@@ -0,0 +1,19 @@
+export OZONE_DIR=$PWD/ozone && \
```

Comment:
There should be some shebang at the start of this file to inform how it must be interpreted.